src/helpers.ts

```diff
@@ -382,24 +382,44 @@ export function loadOrCreateMinter(
     minter.type = minterType.value;
     // populate any minter-specific values
     if (minter.type.startsWith("MinterDALin")) {
-      const contract = IFilteredMinterDALinV1.bind(minterAddress);
-      setMinterExtraMinterDetailsValue(
-        "minimumAuctionLengthInSeconds",
-        contract.minimumAuctionLengthSeconds(),
-        minter
-      );
+      // only do this for legacy minters, because the new minters emit events
+      if (
+        minter.type.endsWith("V0") ||
+        minter.type.endsWith("V1") ||
+        minter.type.endsWith("V2") ||
+        minter.type.endsWith("V3") ||
+        minter.type.endsWith("V4")
```

Comment:
Nit: can probably move this check into a lil util method within this file, given that we use it in a couple places.

Reply:
offloaded to util functions in [1fa1ce1](https://github.com/ArtBlocks/artblocks-subgraph/pull/241/commits/1fa1ce1ceaaf4f54856c7a89942a3b0abaf68946)